graph-manager-docs/source/integrations.md

```diff
@@ -162,24 +161,38 @@ Once you've turned on the integration in Datadog, visit the "Integrations" tab i
 
-All of the metrics reported to Datadog can be notified on directly through Graph Manager via the Notifications feature, but Datadog can be a powerful partner in enabling more complex alerts.
+The [Datadog metric alert query](https://docs.datadoghq.com/api/?lang=curl#metric-alert-query) needed here is
+```
+sum(last_10m):sum:apollo.engine.operations.error_count{service:mygraph,variant:staging}.as_count().rollup(sum).fill(null) / sum:apollo.engine.operations.count{service:mygraph,variant:staging}.as_count().rollup(sum).fill(null) > 0.01
+```
+The `.rollup(sum).fill(null)` is necessary since `apollo.engine.operations.count` is a [Datadog gauge](https://docs.datadoghq.com/developers/metrics/types/), which means it [defaults to using `avg` for time aggregation]() and [defaults to linear interpolation during space aggregation and query arithmetic](https://docs.datadoghq.com/monitors/guide/monitor-arithmetic-and-sparse-metrics/). The `.as_count()` is necessary to ensure that [operation counts are summed before the division and not after](https://docs.datadoghq.com/monitors/guide/as-count-in-monitor-evaluations/).
 
-**Example**: Suppose you have a query that is run against your GraphQL server with a much higher volume in the morning than in the afternoon. You want to enable monitoring on that query's latency and error rates, but if the query volume is very low, you have a higher tolerance for latency and one error will skew the error rate and make the monitor too noisy.
+**Example**: Suppose you're looking at the 95th percentile request latency for some operation, and you notice it follows a cyclical pattern (e.g., it's low during the night, but high during the day). You want to be alerted when this latency deviates from this cyclical pattern, but threshold-based alerts don't easily support this.
 
-> You can use Datadog's [composite monitoring](https://docs.datadoghq.com/monitors/monitor_types/composite/) to enable more complex alerting. You need to start by creating a monitor for each condition you want to track and then combining them in a composite monitor, as explained in the [Datadog documentation](https://docs.datadoghq.com/monitors/monitor_types/composite/).
+You can use [Datadog anomaly detection](https://www.datadoghq.com/blog/introducing-anomaly-detection-datadog/) to be alerted on shifts from that cyclical pattern. You need to specify the metric to observe and the alert conditions/options (e.g. the size of the trigger window), as explained in the [anomaly monitor docs](https://docs.datadoghq.com/monitors/monitor_types/anomaly/).
```

Comment:
how come we're taking out composite monitoring in favor of anomaly detection? Happy to support the change, but I believe the section on composite monitoring was explicitly to point out something that users ask for w.r.t thresholding in our notifications feature.

Reply:
I talked to @zionts offline about this. The main reason I changed it was that the previous example with composite monitoring was confusing, but it turns out the original example was a problem often raised regarding Graph Manager notifications.

I'm down with keeping the original example with composite monitoring, and elaborating on the example to eliminate the confusion.